pulser-core/pulser/channels/dmm.py

```diff
@@ -31,16 +32,20 @@ class DMM(Channel):
     (of zero amplitude and phase). These Pulses are locally modulated by the
     weights of a `DetuningMap`, thus providing a local control over the
     detuning. The detuning of the pulses added to a DMM has to be negative,
-    between 0 and `bottom_detuning`. Channel targeting the transition between
-    the ground and rydberg states, thus encoding the 'ground-rydberg' basis.
+    between 0 and `bottom_detuning`, and the sum of the weights multiplied by
+    that detuning has to be blow `global_bottom_detuning`. Channel targeting
+    the transition between the ground and rydberg states, thus encoding the
+    'ground-rydberg' basis.
 
     Note:
         The protocol to add pulses to the DMM Channel is by default
         "no-delay".
 
     Args:
-        bottom_detuning: Minimum possible detuning (in rad/µs), must be below
-            zero.
+        bottom_detuning: Minimum possible detuning for each atom (in rad/µs),
+            must be below zero.
```

Comment:
Maybe call it `local_bottom_detuning`? Furthermore, this is now just a single value for all qubits, right? So we only have one local bottom detuning and one global bottom detuning and that's it.

Reply:
I thought of keeping `bottom_detuning` for backwards compatibility and because I was afraid `local_bottom_detuning` would be too long, but I guess the first reason does not really matter since it is a patch of the version that introduced that variable name - better to change it now than later. So I am fine with that change - will just imply modifying the schema

Reply:
I will also modify the comment, there has always been only one value for bottom_detuning and global_bottom_detuning

Reply:
I don't think the length is an issue given that you have another variable called `global_bottom_detuning` which is equally long :)